bundles/org.openhab.binding.unifi/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -442,4 +456,75 @@
 		<description>If the access point is enabled</description>
 	</channel-type>
 
+	<channel-type id="devOnline">
+		<item-type>Switch</item-type>
+		<label>Online</label>
+		<description>Online status of the device</description>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="devState">
+		<item-type>String</item-type>
+		<label>Device State</label>
+		<description>The state of the device</description>
+		<state readOnly="true">
+			<options>
+				<option value="0">Offline</option>
+				<option value="1">Connected</option>
+				<option value="2">Pending Adoption</option>
+				<option value="4">Updating</option>
+				<option value="5">Provisioning</option>
+				<option value="6">Unreachable</option>
+				<option value="7">Adopting</option>
+				<option value="9">Adoption Error</option>
+				<option value="10">Adoption Failed</option>
+				<option value="11">Isolated</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="devName">
+		<item-type>String</item-type>
+		<label>Name</label>
+		<description>Name of the device</description>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="devMacAddress">
+		<item-type>String</item-type>
+		<label>MAC Address</label>
+		<description>MAC address of the device</description>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="devIpAddress">
+		<item-type>String</item-type>
+		<label>IP Address</label>
+		<description>IP address of the device</description>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="devUptime">
+		<item-type>Number:Time</item-type>
+		<label>Uptime</label>
+		<description>Uptime of the device (in seconds)</description>
+		<category>Time</category>
+		<state pattern="%.0f %unit%" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="devLastSeen">
+		<item-type>DateTime</item-type>
+		<label>Last Seen</label>
+		<description>Timestamp of when the device was last seen</description>
+		<category>Time</category>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="devExperience">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Experience</label>
+		<description>The average experience of the connected clients</description>
+		<state pattern="%d %unit%" readOnly="true"></state>
+	</channel-type>
+
```

Comment:
Sorry for missing this in first round, but apart from `devState`, these channel types look duplicated. You can use the existing ones and override the label and/or description as needed on channel level. In this case, the label and/or description must also be provided in the update instructions explicitly.

Reply:
No worries, this is great! I did not know that I can override the labels and descriptions but it makes a lot of sense and also answers why I can provide the labels and descriptions in the thing update file :)